sources/meta-ornl/recipes-variscite/images/var-dev-update-full-image.bb

```diff
@@ -8,10 +8,25 @@ require recipes-core/images/ornl-dev-image.bb
 # DART boot is organized for SWUpdate with two rootfs (ping/pong).
 # the kernel, dtb, uboot in a /boot/ folder on both rootfs
 IMAGE_INSTALL_append = " \
-    swupdate \
-    swupdate-www \
-    kernel-image \
-    kernel-devicetree \
-    ksz-initscripts \
-    u-boot-variscite \
+	imx-test \
+	kernel-devicetree \
+	kernel-image \
+	ksz-initscripts \
+	mfgtest \
+	packagegroup-core-full-cmdline \
```

Comment:
Where were these packagegroups located before this?  Were they not being built in?